section2/set9/exercises/01-first-turn-winner/main.go

```diff
@@ -0,0 +1,84 @@
+// Copyright © 2018 Inanc Gumus
+// Learn Go Programming Course
+// License: https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// For more tutorials  : https://learngoprogramming.com
+// In-person training  : https://www.linkedin.com/in/inancgumus/
+// Follow me on twitter: https://twitter.com/inancgumus
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+// ---------------------------------------------------------
+// EXERCISE: First Turn Winner
+//
+//  If the player wins on the first turn, then display
+//  a special bonus message.
+//
+// RESTRICTION
+//  The first picked random number by the computer should
+//  match the player's guess.
+//
+// EXAMPLE SCENARIO
+//  1. The player guesses 6
+//  2. The computer picks a random number, and it happens
+//     to be 6
+//  3. Terminate the game and print the bonus message
+// ---------------------------------------------------------
+
+func main() {
+	const (
+		maxTurns = 5
+		usage    = `Welcome to the Lucky Number Game!
+
+The program will pick %d random numbers.
+Your mission is to guess one of those numbers.
+
+The greater your number is, the harder it gets.
+
+Want to play?`
+	)
+
+	rand.Seed(time.Now().UnixNano())
+
+	args := os.Args[1:]
+
+	if len(args) != 1 {
+		fmt.Printf(usage, maxTurns)
+		fmt.Println()
+		return
+	}
+
+	guess, err := strconv.Atoi(args[0])
+	if err != nil {
+		fmt.Println("Not a number.")
+		return
+	}
+
+	if guess < 0 {
+		fmt.Println("Please pick a positive number.")
+		return
+	}
+
+	for turn := 0; turn < maxTurns; turn++ {
```

Comment:
loop variable should be preferably 1 or 2 letter variable; like t instead-of turn

its mainly for readability and also since loops are very small say 5-10 lines maximum, so the reader and easily remember the context

Reply:
fixed